Modules/ITS/src/ITSFhrTask.cxx

```diff
@@ -38,62 +38,62 @@ ITSFhrTask::ITSFhrTask()
 
 ITSFhrTask::~ITSFhrTask()
 {
```

Comment:
VERY IMPORTANT: please remove any local change from the FHR task. I would not modify the version at P2.

Reply:
of course keep the latest modifications about RunNumberPath and geometry path. Keep maybe a copy of the modified version. Then do `git checkout <filename>` to remove all changes. Then merge the changes from the QC master.

Reply:
ok, I got it.